Lib/asyncio/tasks.py

```diff
@@ -243,8 +243,8 @@ def cancelling(self):
     def uncancel(self):
         """Decrement the task's count of cancellation requests.
 
-        This should be used by tasks that catch CancelledError
-        and wish to continue indefinitely until they are cancelled again.
+        This should be called by the party that called `cancel()` on the task
+        beforehand.
```

Comment:
The previous docstring was invalid, we actively **don't** want for user code to call `uncancel()` when catching CancelledError.